docs/quickstart-aks.md

```diff
@@ -201,6 +201,20 @@ You should also ensure that the `Microsoft.Compute` and `Microsoft.Network` prov
 
 1. Deploy Open Service Broker for Azure on the cluster:
 
+    **Note**
+    Open Service Broker for Azure provides a number of services and each of these services is implemented by a separate module. The stability of individual modules is independent of overall broker stability and is ranked on a scale of `experimental`, `preview`, and `stable`. The broker can be configured to only load modules at or above a specified stability threshold. By default, the helm chart configures the broker to only load modules that are marked as `preview` or `stable`. This currently includes Azure Database for MySQL, Azure Database for PostgreSQL and Azure SQL Database. If you would like to use other services, you will need to add an additional flag to your helm install command:
+
+    ```console
+    helm install azure/open-service-broker-azure --name osba --namespace osba \
+      --set azure.subscriptionId=$AZURE_SUBSCRIPTION_ID \
+      --set azure.tenantId=$AZURE_TENANT_ID \
+      --set azure.clientId=$AZURE_CLIENT_ID \
+      --set azure.clientSecret=$AZURE_CLIENT_SECRET \
+      --set modules.minStability=EXPERIMENTAL
+    ```
+
+    To install OSBA with only `preview` and `stable` services, use the following command.
+
```

Comment:
The sections that follow this fork to show you how to install in bash and in powershell, but this section seems to unconditionally show bash-based instructions. Should do something about that.

Reply:
Also, this is a little repetitive with what follows. Can we have this note here conclude with just instructing users to add `--set modules.minStability=EXPERIMENTAL` to the commands in the following sections _if_ they wish to enable experimental modules? That one line should work fine when appended to either the bash or powershell commands.

Reply:
Ralph's comment was we should put it _before_ so people were familiar with it. Maybe we can just amend the one to have the flag always and they can just change the value?

Reply:
I'm ok with calling this out before. I'm just concerned about the repetition and these instructions not being clear about whether they're for bash or powershell.

Also, I'm realizing now that it's awkward that we cover "here's how to install it with this option" _before_ we cover "here's how to install it."

> Maybe we can just amend the one to have the flag always and they can just change the value?

I think so. Are you thinking we lead with a note about the default and indicate that the value of `modules.minStability` may optionally be modified in the instructions that follow? The instructions that follow can include `--set modules.minStability=PREVIEW` (e.g. equal to the default, but adds some clarity)?

If so... that sgtm.